dev/salt-repo/terraform.tfvars

```diff
@@ -9,6 +10,7 @@ repos = [
   },
   {
     repo_prefix      = "repo/archive/saltstack/salt/linux/"
+    salt_s3_bucket   = "s3"
     salt_s3_endpoint = "https://s3.archive.repo.saltproject.io/"
     yum_prefix       = "yum.defs/saltstack/salt/"
     salt_versions = [
```

Comment:
Did `3003.3` and `3004` listed here under the archive section from the previous PR actually work?  They're not in Salt's archive location.